operators/tokenizer/roberta_tokenizer.cc

```diff
@@ -0,0 +1,251 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Partial code comes from other Microsoft employee.
+
+#include <string>
+#include <vector>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+#include <algorithm>
+#include <list>
+#include <memory>
+#include <regex>
+#include <sstream>
+#include <stdexcept>
+#include <unordered_map>
+#include <functional>
+#include <codecvt>
+#include <mutex>
+
+#include "nlohmann/json.hpp"
+#include "bpetokenizer.hpp"
+#include "string_tensor.h"
+#include "unicode.h"
+
+std::list<std::pair<int, int>> offset_map;
+
+// Note: the following logic comes from CPython: unicodetype_db.h (_PyUnicode_IsWhitespace)
+bool IsWithinUnicodeSpace(char32_t ch) {
```

Comment:
Can we dedup this function?

Reply:
I recall I suggested this when we were cleaning up the CLIP tokenizer code - issue was that each time a tokenizer obj that uses this is built, it contains that primitive already so we get an error like the following: `error LNK2005: "bool __cdecl IsWithinUnicodeSpace(char32_t)" (?IsWithinUnicodeSpace@@YA_N_U@Z) already defined in ocos_operators.lib(gpt2_tokenizer.obj)`, so you suggested to just change its name and keep it in the .cc files for the time being.

Given this, what would be the best way to dedup it now?